### PR TITLE
feat: enable clients(add/remove) and admin(transfer) in crd organization

### PR DESCRIPTION
**Pull request by bjwswang**
Signed-off-by: bjwswang <bjwswang@gmail.com>

Fix: #20

### CODE REVIEW

File: api/v1beta1/organization.go [hunk 1]
```diff
@@ -66,6 +50,32 @@ func (organization *Organization) HasType() bool {
...
+	for _, c := range old {
+		oldMapper[c] = struct{}{}
+	}
+
+	// calculate differences
+	for _, c := range new {
+
+		// added: in new ,but not in old
+		if _, ok := oldMapper[c]; !ok {
+			added = append(added, c)
```
Comment (0xff-dev) [inline api/v1beta1/organization.go:65]: if `c` not int `oldMapper`, we can skip delete step.

**Review by Abirdcfly — APPROVED** [review]
SGTM

should rebase to main to disable gosec test.

**Review by 0xff-dev — APPROVED** [review]
LGTM

**Pull request merged**